AUTHORS.md

```diff
@@ -506,6 +506,7 @@
 |1|Marcus Ramberg <marcus@nordaaker.com>|
 |1|Mark Ide <git@cranstonide.com>|
 |1|Mark Ide <mark@cranstonide.com>|
+|1|Mark Niehe <mniehe@gmail.com>|
```

Comment:
This file is automatically generated once in a while with `make contrib`, no need to edit it